pyprof/prof/linear.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, d):
         if any(x in d.name for x in Linear.gemmKernels):
             self.op_ = "linear"
         else:
-            assert (d.name in Linear.biasKernels)
+            assert any(x in d.name for x in Linear.biasKernels)
```

Comment:
could you explain the logic behind switching from:

- checking if the `d.name` string is a direct match with any string within `Linear.biasKernels`

to:

- checking if any string within `Linear.biasKernels` is contained within the `d.name` string

Reply:
Previously, this code was looking to see if the kernal name was included in Linear.biasKernels. That treats the list of bias kernels as exhaustive. Really, what we want to see is if the kernel name includes the substrings from the bias kernel list. 
This approach is further supported because it is the approach that is taken a few lines above for the gemm kernels.

Reply:
so `d.name` can be longer than the strings in `Linear.biasKernels`? and this condition was observed when we started testing pyprof with different models than the original code must have worked with?

Reply:
Yes `d.name` can be longer than the strings in `Linear.biasKernels`.   
I am not sure what level of testing was run against pyprof before we inherited it. 
We are just starting to get pyprof under test so this is all new behavior that we are trying to capture.